### ISSUE TITLE
Multi Loss Backpropagation Error

### ISSUE THREAD

**Issue by ncalik** [vlfeat/autonn#38]
Hi again,
I'm writing a network which has two loss function. I used pretrained resnet-50. Here I diverted the single feature vector to two loss functions. This is my last layers code: 
![image](https://user-images.githubusercontent.com/17113517/39643452-7dc629e4-4fdd-11e8-9192-beba099b414d.png)

So I've inserted derOutput like that : 
![image](https://user-images.githubusercontent.com/17113517/39643587-ef15735c-4fdd-11e8-8c1e-c720cbf3f8c5.png)

When I evaluate net, I get below error : 

![image](https://user-images.githubusercontent.com/17113517/39643535-bd77e10e-4fdd-11e8-9889-47663194fa94.png)

Is this error due to the coincidence on the single attribute vector of the two backward derivations?

**Comment by ncalik** [comment]
Solved !! 
I should use **{**1,1,0,0**}**
